File: PHP/7php 1/angle.php
<?php
/*
|--------------------------------------------------------------------------
| Sum of angles
|--------------------------------------------------------------------------
|
*/

function angle(int $n): int {
   return ($n - 2) * 180;
}


function angle(int $n): int {
   $res = 0;
   for ($i = 3; $i <= $n; $i++) {
     $res += 180;
   }
   return $res;
}


function angle(int $n): int {
   return (2 * $n - 4) * 90;
}


function angle(int $n): int {
   return 180 + ($n - 3) * 180;
}


print_r(angle(3) . ' ' . 180 . '
');
print_r(angle(4) . ' ' . 360 . '
');
print_r(angle(5) . ' ' . 540 . '
');

File: Python/7py 1/angle.py
"""
|--------------------------------------------------------------------------
| Sum of angles
|--------------------------------------------------------------------------
|
"""

def angle(n):
   return (n - 2) * 180


def angle(n):
   return (n+~1)*180


def angle(n):
   a = 360 / n
   b = 180 - a
   return round(b * n)


def angle(n):
   return max([(n-2)* 180])


print(angle(3), 180)
print(angle(4), 360)